api/PositionOptions.json

```diff
@@ -0,0 +1,540 @@
+              "version_added": "5",
+              "version_removed": true,
+              "notes": "Secure context not required"
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": true
+            },
+            {
+              "version_added": true,
+              "version_removed": true,
+              "notes": "Secure context not required"
+            }
+          ]
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "enableHighAccuracy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PositionOptions/enableHighAccuracy",
+          "support": {
+            "webview_android": [
+              {
+                "version_added": "47"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "47",
+                "notes": "Secure context not required"
+              }
+            ],
+            "chrome": [
+              {
+                "version_added": "47"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "47",
+                "notes": "Secure context not required"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "47"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "47",
+                "notes": "Secure context not required"
+              }
+            ],
+            "edge": [
+              {
+                "version_added": null
+              },
+              {
+                "version_added": true,
+                "notes": "Secure context not required"
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": null
+              },
+              {
+                "version_added": true,
+                "notes": "Secure context not required"
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "55"
+              },
+              {
+                "version_added": "3.5",
+                "version_removed": "55",
+                "notes": "Secure context not required"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "55"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "55",
+                "notes": "Secure context not required"
+              }
+            ],
+            "ie": {
+              "version_added": "9",
+              "notes": "Secure context not required"
+            },
+            "opera": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "16",
```

Comment:
Opera's the weird one here. The story for this property is:

* `"version_added": "16"`
* `"version_added:" "10", "version_removed": "15"`